src/main/scala/afenton/bazel/bsp/BuildMetaData.scala

```diff
@@ -1,6 +1,6 @@
 package afenton.bazel.bsp
 
 object BuildMetaData {
```

Comment:
maybe you want to use this:

https://github.com/sbt/sbt-buildinfo

Reply:
Will tackle in follow up PR.